spec/features/lateness_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
```

Comment:
I'm not completely clear what is being tested here. Are we saying that, if you're late, you can fill in a reason? If so, shouldn't we also have a test saying that if you're not late, you can't fill in a reason?

Does `expect(page).to have_current_path(root_path)` really test anything here?